content/en/load-testing/Quick Start/40-run-a-test.md

```diff
@@ -10,50 +10,50 @@ description: >
 
 ## Setup a Test Execution
 
-To run a test, first tell your setup (`<testsuite>\config\project.properties`) which test configuration to use:
+Ok, so now you have the [demo shop running on your machine](../20-demo-application/) and you have [set up the demo testsuite](../30-demo-test-suite/). To run a test, first tell your setup (`<posters-simple-loadtest-suite>\config\project.properties`) which test configuration to use and where to find your posters demo shop instance:
 
 ```bash
 com.xceptance.xlt.testPropertiesFile = my-test.properties
+
+store-url = https://localhost:8443/posters/
 ```
 
-Then you can configure the basic properties of the test suite and the load profile (in `<testsuite>\config\my-test.properties`). Our demo test suite comes with basic test.properties which you can use as a starting point for whatever you are planning.
+Then you can configure the basic properties of the test suite and the load profile (in `<posters-simple-loadtest-suite>\config\my-test.properties`). Our demo test suite comes with basic test.properties which you can use as a starting point for whatever you are planning.
```

Comment:
Assume that most developers used Mac or Linux and avoid \ here, because that might be easier to understand.